README.md

```diff
@@ -31,13 +136,231 @@ const client = sanityClient({
 export default client
 ```
 
-`const client = sanityClient(options)`
+#### [CommonJS]
+
+```js
+const {createClient} = require('@sanity/client')
+
+const client = createClient({
+  projectId: 'your-project-id',
+  dataset: 'bikeshop',
+  apiVersion: '2021-03-25', // use current UTC date - see "specifying API version"!
+  token: 'sanity-auth-token', // or leave blank for unauthenticated usage
+  useCdn: true, // `false` if you want to ensure fresh data
+})
+
+module.exports = client
+```
+
+#### [TypeScript]
+
+```ts
+import {createClient, type ClientConfig} from '@sanity/client'
+
+const client: ClientConfig = {
+  projectId: 'your-project-id',
+  dataset: 'bikeshop',
+  apiVersion: '2021-03-25', // use current UTC date - see "specifying API version"!
+  token: 'sanity-auth-token', // or leave blank for unauthenticated usage
```

Comment:
I've updated these examples to just not use it as they're meant to be MVP snippets anyway.

So now only the first example mentions it, and perhaps that one should link to a docs page that explains a bit more what it's for? https://github.com/sanity-io/client/tree/modernize#quickstart

Especially since we have both viewer and editor tokens and we have 3 main use cases for it (from our users perspective):
- write tokens: mutate datasets, uploads assets
- viewer tokens for accessing private datasets
- viewer tokens for `@sanity/groq-store` (`next-sanity` and `@sanity/preview-kit`'s preview mode logic) to do `overlayDrafts`

Reply:
cc @kmelve

Reply:
Good point. I kinda wanted to make it very clear in the quickstart that the client isn't only for querying documents, but also for updating them. 

Btw. have we thought about using `withCredentials` for preview stuff, or does that get messy because we also need server-side view access?

Reply:
> Btw. have we thought about using `withCredentials` for preview stuff, or does that get messy because we also need server-side view access?

`withCredentials` is becoming less of a thing you can depend on since more browsers are going cookieless

Reply:
Yeah all our new templates with `live-preview` setup use a `token` due to the unreliability of `withCredentials`.

@kmelve I love the idea of using this line
```
// token: process.env.SANITY_SECRET_TOKEN // Only if you want to update content with the client
```
to bust the assumption that Studio is where you write, and `@sanity/client` is where you read.
So what do you think about expanding the pitch and adding a link?
```
// With a token you can query unpublished drafts, update content, upload images and files.
// https://www.sanity.io/docs/keeping-your-data-safe
// token: process.env.SANITY_SECRET_TOKEN
```
That's the best link I could find, but you get the idea :) 

Btw it looks like [these docs](https://www.sanity.io/help/js-client-usecdn-token) aren't updated with the the [v3 fix?](https://github.com/sanity-io/client/blob/371515b9c517c1849fcd2fc214872fb7fa7e7061/CHANGELOG.md?plain=1#L108)